robot_driver/include/robot_driver/robot_driver.h

```diff
@@ -313,6 +347,15 @@ class RobotDriver {
   /// Leg Controller template class
   std::shared_ptr<LegController> leg_controller_;
 
+  /// State Estimator template class
+  std::shared_ptr<StateEstimator> state_estimator_;
+
+  /// Complementary Filter class
+  std::shared_ptr<CompFilterEstimator> comp_filter_estimator_;
+
+  /// EKF Estimator template class
+  std::shared_ptr<EKFFilterEstimator> ekf_estimatror_;
```

Comment:
Don't use these, everything should be called off of the state_estimator_ pointer (there should only ever be one), see controller setup for comparison.